wweval/R/ms_fig4.R

```diff
@@ -166,12 +166,16 @@ make_fig4_admissions_overall <- function(eval_hosp_data,
     dplyr::group_by(date) |>
     dplyr::summarise(
       total_hosp = sum(daily_hosp_admits)
+    ) |>
+    dplyr::mutate(
+      date = date
```

Comment:
I don't think this accomplishes what you want. `summarise` shouldn't drop `date` (since you grouped by `date`). But if the dataframe being passed does _not_ have a column named date, this mutate call will either:
1. error
2. create a column named `date` whose value is whatever was in a variable named `date`, if one exists in the environment.

```
library(tibble)
library(dplyr)
library(lubridate)

tibble(a = 1:5) |>
   mutate(date = date) # error

date <- "2024-01-01"
tibble(a = 1:5) |>
   mutate(date = date) # adds a column where all values of date are "2024-01-01"

tibble(a = 1:5, date=seq(lubridate::ymd("2024-01-01"), length.out=5, by="day") |>
   mutate(date = date) # does nothing
```